src/BatteryModel.cpp

```diff
@@ -454,10 +454,18 @@ PrognosticsModel::predicted_output_type BatteryModel::predictedOutputEqn(double,
     double qnB = x[indices.states.qnB];
 
     auto z_new = getPredictedOutputVector();
-    z_new[PRED_OUT::SOC] = (qnS + qnB) / parameters.qnMax;
+    z_new[EVENT_STATE::SOC] = (qnS + qnB) / parameters.qnMax;
     return z_new;
```

Comment:
PredictedOutputEqn should just return getPredictedOutputVector() (an empty vector of size 0)